website/versioned_docs/version-1.0.0/runtime/initializing-storage.md

```diff
@@ -10,18 +10,18 @@ In some cases storage should be initialized with a value before any user has int
 
 ## Hard-Code Default Values
 
-The simplest way (and one that technically does not use genesis configuration) to initialize storage is simply to hard-code default values for each parameter. This option is best when there are clear default values that all deployments of the module will want to use. The runtime developer can choose this option by including `config()` in the `decl_storage!` marco, and an initial value at the end of the line.
+The simplest way (and one that technically does not use genesis configuration) to initialize storage is simply to hard-code default values for each parameter. This option is best when there are clear default values that all deployments of the module will want to use. The runtime developer can choose this option by including in the `decl_storage!` macro an initial value at the end of the line.
 
 Consider this example module where players ante to play a game and add to the pot by raising.
 
 ```rust
 decl_storage! {
   trait Store for Module<T: Trait> as MyModule {
     // Points ante'd to play the game
-    pub Ante get(ante) config(): u32 = 5;
+    pub Ante get(ante): u32 = 5;
 
     // Minimum raise on each turn
-    pub MinRaise get(min_raise) config(): u32 = 7;
+    pub MinRaise get(min_raise): u32 = 7;
   }
 }
 ```
```

Comment:
on the line below (29) maybe we can change `it will revert to the default.` to `it will revert to this specified default.` to be more precise ?

Reply:
fixed